index.html

```diff
@@ -4,9 +4,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" />
 </head>
 <body>
+    <section>
+        <h1 id="titleCityName">Weather Report for Bridgewater</h1>
+    </section>
+
+    <section class="main-container">
+
+        <section>
+            <h2>Temperature</h2>
+            <button id="increaseTemp" class="big-text">Click to increase</button>
+            <h1 id="tempDisplay"><p class="text-color-blue">Current Temperature: 80 F</p></h1>
+            <button id="decreaseTemp" class="big-text">Click to decrease</button>
+            <h2>City</h2>
+            <section>
+                <input id="cityInput" type="text">
+                <button id="enterCityName" class="big-text">Type city name then click</button>
+                <button id="resetCity" class="big-text">Click to reset to default city</button>
+            </section>
+            <h2>Sky</h2>
+            <section>
+                <select name="Sky" id="sky-select">
+                    <option value="">--Please choose an option--</option>
+                    <option value="sun">Sunny</option>
+                    <option value="cloud">Cloudy</option>
+                    <option value="rain">Rainy</option>
+                    <option value="snow">Snowy</option>
+                </select>
+            </section>
+        </section>
     
+        <section>
+            <p id="sky">
+                <img alt="Nice Sky" src="assets/sky-sun.jpeg">
+            </p> 
```

Comment:
```suggestion
                <img id="sky" alt="Nice Sky" src="assets/sky-sun.jpeg">
```
You may have to change some CSS, but if we want to change values in an HTML element, we should isolate the direct element, not the parent.